neurodamus/utils/memory.py

```diff
@@ -99,3 +99,27 @@ def get_mem_usage():
     usage_mb = float(data_size) * os.sysconf("SC_PAGE_SIZE") / 1024 ** 2
 
     return usage_mb
+
+
+class SynapseMemoryUsage:
+    ''' A small class that works as a lookup table
+    for the memory used by each type of synapse.
+    The values are in KB. The values cannot be set by the user.
+    '''
+    def __init__(self):
+        self._synapse_memory_usage = {
+            "ProbAMPANMDA": 1.7,
+            "ProbGABAAB": 2.0,
+            "Gap": 2.0,
+            "Glue": 0.5
+        }
+
+    def __getitem__(self, item):
+        return self._synapse_memory_usage[item]
+
+    def __setitem__(self, key, value):
+        raise ValueError("Cannot set values in SynapseMemoryUsage. "
+                         "Values are hardcoded.")
```

Comment:
I would drop these two methods. If they are not implemented then `[]` operator is not enabled at all.

Reply:
Makes sense. Removed them.